src/meta.php

```diff
@@ -86,6 +86,26 @@ function register_meta_helper(
 		$meta_key
 	);
 
+	// Allow setting meta for all of an object type.
+	if (
+		(
+			is_array( $object_slugs ) &&
+			count( $object_slugs ) === 1 &&
+			'all' === $object_slugs[0]
+		) ||
```

Comment:
I wonder if it's worth considering `['all']` to be a valid value here. this condition would return true with `['all']` and `['all', 'a-valid-slug']` which is a little confusing.

would it be worth making it apply to all object types if `$object_slugs` is equal to `'all'` vs supporting an array with `all` being the first value? It would make this code a bit cleaner but I see both arguments for it.

Reply:
@srtfisher Just to confirm, I made [this 3v4l](https://3v4l.org/Isj1l)